src/__tests__/VoucherRedemption.tsx

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import axios from 'axios';
+import { renderIntegration } from '../utilities/testing/render';
+import VoucherPage from '../pages/VoucherRedemption';
+import { cleanup, fireEvent } from '@testing-library/react';
+import {
+  VoucherProvider,
+  VoucherTypes,
+} from '../utilities/hooks/VoucherContext';
+
+jest.mock('../utilities/api/interactionManager', () => {
+  const shunfaBakeryResponse: any = {
```

Comment:
is it possible to pull this out into a separate json file called sellersResponse.json

Reply:
yeah i was planning on making a whole mocks directory containing all different types of mock responses. When i add more tests to the other files im going to create that.